app/controllers/wikis_controller.rb

```diff
@@ -0,0 +1,55 @@
+class WikisController < ApplicationController
+  def index
+    @wikis = Wiki.all
+    @wiki = Wiki.new
+  end
+
+  def show
+    @wiki = Wiki.find(params[:id])
+  end
+
+  def create
+    @wiki = Wiki.new(wiki_params)
```

Comment:
You aren't associating the wiki with the user here. Missed that in my original review.